python/ws_stomp.py

```diff
@@ -5,11 +5,12 @@
 
 
 class WebSocketStomp:
-    def __init__(self, ws_url, user, password, ssl_ctx):
+    def __init__(self, ws_url, user, password, ssl_ctx, filter):
```

Comment:
Let's put filter param in stomp_subscribe instead. Because it is related to the topic

Reply:
moved to stomp_subscribe